src/content-helper/editor-sidebar/performance-details/performance-details.scss

```diff
@@ -0,0 +1,367 @@
+@import "../../common/css/variables";
+@import "../../common/css/functions";
+
+.wp-parsely-content-helper .wp-parsely-performance-panel {
+	padding: calc(16px);
+	border-top: 1px solid rgb(221, 221, 221);
+	margin-top: -1px;
+
+
+	/** View in Parse.ly button */
+	.components-button.wp-parsely-view-post {
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		width: 100%;
+	}
+
+	.panel-body {
+		width: 100%;
+	}
+
+	.performance-stat-panel {
+		display: flex;
+		padding: to_rem(6px) 0 to_rem(16px) 0;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: to_rem(8px);
+		align-self: stretch;
+
+		.panel-header {
+			display: flex;
+			height: to_rem(16px);
+			width: 100%;
+			align-items: center;
+			gap: to_rem(6px);
+			margin-bottom: 0;
+
+			/** Level 2 heading */
+			&.level-2 {
+				margin-bottom: to_rem(15px);
+
+				h2 {
+					margin-bottom: 0;
+				}
+			}
+
+			/** Level 3 heading */
+			&.level-3 {
+
+				h3 {
+					margin-bottom: 0;
+					line-height: to_rem(16px);
+					font-size: var(--font-size--smaller, to_rem(11px));
+					font-style: normal;
+					font-weight: 600;
+					text-transform: uppercase;
+				}
+			}
+		}
+
+		.panel-subtitle {
+			display: flex;
+			flex: 1 0 0;
+			height: to_rem(16px);
+			align-items: center;
+			gap: to_rem(6px);
+			align-self: stretch;
+			color: var(--Gutenberg-Gray-700, #757575);
+
+			/* Label */
+			font-size: var(--font-size--smaller, to_rem(11px));
+			font-style: normal;
+			font-weight: 600;
+			line-height: to_rem(16px); /* 145.455% */
+			text-transform: uppercase;
+		}
+	}
+
+	.panel-settings {
+
+		.components-select-control {
+			display: flex;
+			align-items: flex-start;
+			gap: to_rem(16px);
+			align-self: stretch;
+		}
+
+		.components-input-control__prefix {
+			margin-left: to_rem(8px);
```

Comment:
@vaurdan @acicovic I up-streamed this fix to core but they do provide a wrapper for prefixes and suffixes to apply the correct padding. It's mentioned in the storybook entry for inputControl, which is apparently what the selectControl is based on.